src/metabase/api/setup.clj

```diff
@@ -103,83 +110,105 @@
 
 ;;; Admin Checklist
 
+(defmulti ^:private admin-checklist-entry
+  {:arglists '([entry-name])}
+  identity)
```

Comment:
Since this is private (ie not extendable) does it make sense to use multimethods over a static map?

Reply:
I did it mostly for clarity, I think this is still a bit clearer than a static map